addon/components/pix-checkbox.js

```diff
@@ -22,7 +22,13 @@ export default class PixCheckbox extends Component {
   }
 
   get isDisabled() {
-    return this.args.isDisabled || this.args.disabled;
+    warn(
+      'PixCheckbox: @isDisabled attribute should be a boolean, a string was passed. Checkbox is considered disabled.',
+      ['false', 'null', 'undefined'].includes(this.args.isDisabled),
```

Comment:
*thought :* `![false,true].includes(this.args.isDisabled)` serait plus clair. parce que l'on pourrait passer `toto` et le warn ne serait pas executé

Reply:
Yes, j'ai aussi ajouté `null` et `undefined` car ça générait des warnings dans les cas de figure : 
- `<PixCheckbox>...</PixCheckbox>` (=> `undefined`)
- `<PixCheckbox @isDisabled={{undefined}}>...</PixCheckbox>`
- `<PixCheckbox @isDisabled={{null}}>...</PixCheckbox>` Je pense que c'est plus permissif, WDYT ?